compiler/modules/front.syntax/ast.bal

```diff
@@ -453,6 +454,7 @@ public type FunctionTypeDesc record {|
     // XXX need to handle rest public type
```

Comment:
Not completely sure what this XXX is for. We need to figure it out and deal with it,